start.py

```diff
@@ -715,6 +734,43 @@ def _configure_kudu(deployment, cluster, kudu_version):
             # Only one parcel can be activated at a time, so once one is found, our work is done.
             break
 
+
+def _validate_kudu_version(kudu_version, cdh_version, cm_cluster):
+    cdh_version_tuple = tuple([int(i) for i in cdh_version.split('.')])
+
+    if cdh_version_tuple < EARLIEST_CDH_VERSION_WITH_KUDU:
+        cdh_version_from_map = KUDU_PARCEL_VERSIONS.get(kudu_version)
+        if cdh_version_from_map is None:
+            raise Exception('kudu-version {} is not available for '
+                            'CDH {} Aborting ...'.format(kudu_version, cdh_version))
+    else:
+        # For CDH 5.13.0 and above, Kudu is integrated in CDH and only the integrated version can be installed.
+        # Hence check if the parameter kudu_version matches integrated kudu_version for the specific CDH version.
+        cluster_info = cm_cluster.get_cluster_info()
+        cluster_kudu_version = None
```

Comment:
Setting this to `None` seems not very Pythonic. Instead, why not just have the for loop and then put in an `else` to raise an Exception if Kudu is never found?

Reply:
Done.